.github/workflows/build.yml

```diff
@@ -4,12 +4,12 @@ on:
   push:
     branches: [dev]
     paths-ignore:
-      - "docs/**"
-      - "readme.md"
+      - "Docs/**"
+      - "Readme.md"
 
   pull_request:
     branches: [dev]
 
 jobs:
   call-build-workflow:
-    uses: deanmarcussen/NuGet-Publishing-GitHub-Actions/.github/workflows/build.yml@v1
+    uses: Lombiq/NuGet-Publishing-GitHub-Actions/.github/workflows/build.yml@v1

```

Comment:
Any reason why this isn't `@dev`? Also in the publish action and in HE.

Reply:
It's tag based not branch based. As far as I could tell referencing actions supports commit hash or tag, not branch name.
I decided to prefer tags over commit hashes, as tags are movable (meaning you can make non breaking changes to the workflow, and move the tag)

https://docs.github.com/en/actions/learn-github-actions/reusing-workflows#calling-a-reusable-workflow

Reply:
It can also reference a branch: https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#example-using-versioned-actions

Reply:
That's actions, not callable workflows (two different things).

Might still work. I'll have to test it, and let you know.

Reply:
I see. Judging from this example, it's the same for workflows too: https://docs.github.com/en/actions/learn-github-actions/reusing-workflows#passing-inputs-and-secrets-to-a-reusable-workflow

Reply:
You're right! Done for both